docs/source/tutorials.md

```diff
@@ -1,32 +1,21 @@
 # Tutorials
 
-```{note}
-This is a work in progress; visit again soon for updated content!
-```
-
-The following tutorials are meant as an intro to the full stack:
+The following tutorials are meant as an introduction to the full stack:
 
 ```{toctree}
-:maxdepth: 2
+:maxdepth: 1
 
 getting_started_with_jax_for_AI
 digits_vae
-JAX_for_PyTorch_users
-JAX_porting_PyTorch_model
 digits_diffusion_model
-JAX_for_LLM_pretraining
-JAX_basic_text_classification
-JAX_examples_image_segmentation
-JAX_Vision_transformer
-JAX_machine_translation
 JAX_visualizing_models_metrics
-JAX_image_captioning
-JAX_time_series_classification
-JAX_transformer_text_classification
-data_loaders_on_cpu_with_jax
-data_loaders_on_gpu_with_jax
+data_loaders
+JAX_for_PyTorch_users
```

Comment:
We could think about putting the two pytorch tutorials under their own heading – what do you think?

Reply:
I think that's a good idea, will do!